recipe/meta.json

```diff
@@ -13,6 +13,7 @@
       "jq =1.6.*",
       "jsonschema =4.17.*",
       "lxml =4.9.*",
+      "make >=3.8",
```

Comment:
This was added automatically due to the change to `meta.yaml`, below -- see the comment there.